mobile/Features/CheckIn/Sleep.jsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react';
+import {
+  Text, View, Pressable, Image,
+} from 'react-native';
+import { useRoute } from '@react-navigation/native';
+import Slider from '@react-native-community/slider';
+import ProgressBar from 'react-native-progress/Bar';
+import styles from './SleepStyle';
+
+function Sleep() {
+  const route = useRoute();
+  // update progress
+  const numPages = route.params?.numPages;
+  const progress = 2 / numPages;
+  // slider state
+  const [slider, setSlider] = useState(0);
+
+  const onSliderChange = (sliderValue) => {
+    setSlider(sliderValue);
+  };
+
+  const continueButton = () => {
+    // console.log('continue');
+  };
+
+  const skipButton = () => {
+    // console.log('skip');
+  };
```

Comment:
if it's not too much work, could you make it so that both of these navigate back to the landing page? Avoiding dead ends in our app navigation is helpful for saving time when debugging and for future work/demos!